index.html

```diff
@@ -1,16 +1,101 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Responsive Cake webpage</title>
     <!-- Add a link to your css file here -->
-</head>
+    <link rel = stylesheet type= text/css href="style.css"/>
+  </head>
+  <body>
+    <header class="head">
+      <figure>
+        <img class="logo" src="images/sport-car.jpg" alt="car-logo" />
+        <figcaption class="contact-info">
+          <p>Contact us</p>
+        </figcaption>
+      </figure>
+      <nav class="nav-menu">
+        <ul class="list">
+          <li class="list-item">Home</li>
+          <li class="list-item">Collection</li>
+          <li class="list-item">About</li>
+          <li class="list-item">Service</li>
+          <li class="list-item">Contact</li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main-container">
+      <aside class="sidebar">
+        <h1 class="title">Hello</h1>
+        <p class="info">information</p>
+      </aside>
 
-<body>
-    <!-- Add your markup here -->
-</body>
+      <section class="main-img-container">
+        <img class="main-image" src="images/vw beetle.jpg" alt="vw-beetle" />
+      </section>
```

Comment:
You should wrap these 2 elements (sidebar and main-img-container) inside flex container so that the sidebar and the main image appear in a row without using `float: left / right`
Add a section around these and set it to display: flex